### PR TITLE
Gsa/production release

### PR DESCRIPTION
**Pull request by felder101**
Issue https://github.com/GSA/smartpay-training/issues/456 GSPC | Email Title and Content Once User Initially Attempts to Begin GSPC Verification
Issue https://github.com/GSA/smartpay-training/issues/419 Administrator | Loading GSPC Eligible List into the Training System
Issue https://github.com/GSA/smartpay-training/issues/494 Create Training - Admin Landing Page
Issue https://github.com/GSA/smartpay-training/issues/496 Dependabot Alert: FastAPI Content-Type Header ReDoS

### CODE REVIEW

File: training/schemas/gspc_invite.py [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+class GspcInvite(BaseModel):
+    email_addresses: str
+    certification_expiration_date: datetime
+    valid_emails: Optional[List[str]] = []
+    invalid_emails: Optional[List[str]] = []
+
+    @field_validator('certification_expiration_date')
+    @classmethod
+    def check_certification_expiration_date(cls, value):
+        value is datetime
```
Comment (mark-meyer) [inline training/schemas/gspc_invite.py:16]: This seems a little strange — just an expression doing nothing. Maybe it was meant to be deleted or should be a comment?

**Review by mark-meyer — COMMENTED** [review]
There's one small thing noted in a comment — just looks like leftover code that should probably be deleted. But lets fix that before pushing.

It looks like the Vue components are still in progress, but it we don't have tests for new components yet. The coverage report won't tell you about new code that don't have a .spec file, so the front-end coverage report will be a little too forgiving. Lets not let this hold up this PR, but we should have test for Vue components as well as the python tests (which look nice here) as this develops. There should be a lot of go-bys in the `training-front-end/src/components/__tests__` directory.

File: training/schemas/gspc_invite.py (hunk 1 above)
Comment (felder101) [reply training/schemas/gspc_invite.py]: Nice catch. Code was used for debugging. Redundant code has been removed.

**Review by mark-meyer — APPROVED** [review]
LGTM.

**Pull request merged**